components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/storage/model/EventSpecificationEntity.java

```diff
@@ -26,6 +30,13 @@
     @Column(name = "spec_data", length = Integer.MAX_VALUE)
     private String specData;
 
+    @EdmProtectedBy(name = "tenant_id")
+    @EdmIgnore
+    @Column(name = "tenant_id", length = 256)
+    @Convert("uuidConverter")
+    @TypeConverter(name = "uuidConverter", dataType = Object.class, objectType = UUID.class)
+    private UUID tenant;
+
```

Comment:
Is those needed since eventSpecifications are not part of the OData API and the tenancy is based on the repository method?